### PR TITLE
shell: Add stylesheet, round off the user selection corners

### PR DESCRIPTION
**Pull request by samcday**
see [ #22 ](https://github.com/samcday/phrog/issues/22#issuecomment-2323463859)

### CODE REVIEW

File: resources/phrog.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+phosh-lockscreen list {
```
Comment (agx) [inline resources/phrog.css:1]: Adding

```
gtk_widget_class_set_css_name (widget_class, "phrog-user-session-page"); 
```

and then use that as namespace for your CSS changes would ensure CSS changes on phosh's lockscreen  don't break the geeters CSS.

**Comment by agx** [comment]
Looks nicer!

File: resources/phrog.css (hunk 1 above)
Comment (samcday) [reply resources/phrog.css:1]: Thanks, I think c73fff37 is better now?

Comment (agx) [reply resources/phrog.css:1]: Thanks! That looks good to me (the name basically doesn't matter), i'd just stay clear of the `phosh-` namespace so you don't accidentally override any styles. 

As you still load phosh's default style (which I think is good so things like high contrast can work out of the box) we might still accidentally break things but it becomes way less likely that way.

Comment (samcday) [reply resources/phrog.css:1]: > i'd just stay clear of the phosh- namespace so you don't accidentally override any styles.

Makes sense, and agreed.

> As you still load phosh's default style ...

Yeah, I was wondering that to myself also. We'll see soon enough :D

**Pull request merged**